src/main/java/org/takes/facets/auth/codecs/CcAES.java

```diff
@@ -145,15 +118,40 @@ public Identity decode(final byte[] bytes) throws IOException {
     }
 
     /**
+     * This method will create AlgorithmParameterSpec with the block size.
```

Comment:
@sebing here too, writing `this method will` is unnecessary, `Creates` is enough